index.bs

```diff
@@ -957,6 +972,8 @@ a [=trigger state=] |triggerState|:
     :: null
     : [=attribution trigger/debug reporting enabled=]
     :: false
+    : [=attribution trigger/aggregation coordinator=]
+    :: "<code>[=aggregation coordinator/aws-cloud=]</code>"
```

Comment:
Shouldn't this be an implementation-determined default?

Reply:
Updated to use null as default and implementation can determine what to do with it.